gtk/gtk-play.c

```diff
@@ -55,6 +59,9 @@ typedef struct
   GtkWidget *media_info;
   gulong seekbar_value_changed_signal_id;
   gboolean playing;
+  GdkPixbuf *cover_pixbuf;
+  GstSample *cover_sample;
+  GMutex lock;
```

Comment:
play->cover_pixbuf is accessed by both gtk_widget_queue_draw() and media_info_updated_cb ()  hence I was using mutex. The idea is media_info_updated_cb() updates the pixbuf and draw function draw's the updated picture.